src/corporacreator/preprocessors/fr.py

```diff
@@ -8,5 +67,9 @@ def fr(client_id, sentence):
     Returns:
       (str): Cleaned up sentence. Returning None or a `str` of whitespace flags the sentence as invalid.
     """
-    # TODO: Clean up fr data
-    return sentence
+    text = maybe_normalize(sentence, mapping=FR_NORMALIZATIONS + [REPLACE_SPELLED_ACRONYMS])
+    text = replace_numbers(text, locale='fr', ordinal_regex=FIND_ORDINAL_REG)
```

Comment:
Just wondering if we should do that now or later: as you shown, my heuristics were not perfect, so maybe it'd be best if I listened to recording and adjust with `client_id`, and fix Common Voice dataset at the same time ?

Reply:
As far as I can tell, it work just fine as is (I am also using it in trainingspeech projet).
It is a good idea to pick / listen a few examples to check but checking all the examples will take a lot of time...
Personally, I won't have such bandwidth anytime soon...

Reply:
That's why I was suggesting that I do it :)

Reply:
@lissyx why not doing it in another PR ?

Reply:
@nicolaspanel That's what was implied :)

Reply:
@lissyx @nicolaspanel This is related to my "invalidate all sentences with digits" comment above.

I'd be interested in your take on the Draconian idea to have [`common.py`](https://github.com/mozilla/CorporaCreator/blob/master/src/corporacreator/preprocessors/common.py) mark as invalid any sentences with digits in them.

Reply:
@kdavis-mozilla I guess it's not such a bad idea, with a logging mode to help identify and fix the dataset as well.